node/src/server-modules/GlideJson.ts

```diff
@@ -511,4 +511,84 @@ export class GlideJson {
 
         return _executeCommand<ReturnTypeJson<number>>(client, args);
     }
+
+    /**
+     * Increments or decrements the JSON value(s) at the specified `path` by `number` within the JSON document stored at `key`.
+     *
+     * @param client - The client to execute the command.
+     * @param key - The key of the JSON document.
+     * @param path - The path within the JSON document.
+     * @param num - The number to increment or decrement by.
+     * @returns
+     *     - For JSONPath (path starts with `$`):
+     *       - Returns a bytes string representation of an array of bulk strings, indicating the new values after incrementing for each matched `path`.
```

Comment:
```suggestion
     *       - Returns a string representation of an array of strings, indicating the new values after incrementing for each matched `path`.
```
update below too
you can use `GlideString` term if you like